run: exit status 254
/tmp/call-a-foreign-language-function.dart:15:6: Error: 'main' is already declared in this scope.
void main() {
     ^^^^
/tmp/call-a-foreign-language-function.dart:9:6: Context: Previous declaration of 'main'.
void main() {
     ^^^^
/tmp/call-a-foreign-language-function.dart:16:3: Error: Can't use 'main' because it is declared more than once.
  main();
  ^
